.github/workflows/release-please.yml

```diff
@@ -132,3 +133,44 @@ jobs:
               config/rendered/release.yaml
               config/samples/end-to-end.yaml
               open-feature-operator-sbom.spdx.json
+
+  release-charts:
+    needs: release-please
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
+    if: ${{ needs.release-please.outputs.release_created }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          ref: ${{ env.GITHUB_PAGES_BRANCH }}
+
+      - name: Setup go
+        uses: actions/setup-go@v3
+        with:
+          go-version: ${{ env.DEFAULT_GO_VERSION }}
+
+      - name: Configure Git
+        run: |
+          git config user.name "$GITHUB_ACTOR"
+          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+
+      - name: Merge release to gh-pages
+        run: git merge ${{ needs.release-please.outputs.release_tag_name }}
+
+      - name: Generate helm charts
+        run: make helm-package
```

Comment:
Do we need to define the image for the controller here? 
The makefile commands use the IMG argument which defaults to `controller:latest`
I may have missed something here though